platform/src/main/java/org/stellar/anchor/platform/utils/RpcUtil.java

```diff
@@ -16,7 +16,11 @@ public class RpcUtil {
   public static final String JSON_RPC_VERSION = "2.0";
 
   public static RpcResponse getRpcSuccessResponse(Object id, Object result) {
-    return RpcResponse.builder().jsonrpc(JSON_RPC_VERSION).id(id).result(result).build();
+    return RpcResponse.builder()
+        .jsonrpc(JSON_RPC_VERSION)
+        .id(id)
+        .result(id != null ? result : null)
```

Comment:
BTW, I think id should be mandatory cc @JakeUrban

Reply:
You are right. Recently Jake provided us details of how SorobanRPC is developed.
In this PR I made id field mandatory. In next PR we will make changes in response to include transaction id in error/result